### PR TITLE
Setlists

### PR DESCRIPTION
**Pull request by anniviz**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -65,13 +73,55 @@ function App() {
...
+          : 'no song'}
+      </SwipeableViews>
+    )
+  }
+
+  function handleSwipeableViewWithAllSongs(songs) {
+    swipeableViewContent = isLoading ? (
+      'Loading ...'
+    ) : (
+      <SwipeableViews
```
Comment (srmnq) [inline src/App.js:88]: is there a chance to reduce those two functions into one? they look quite alike?!

File: src/SideList/SideList.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+      </SideListWrapper>
+      <ListMenu>
+        <MenuItem
+          style={{ borderRadius: '0 0 0 12px' }}
+          onClick={handleSetlistClick}
+        >
+          <img
+            className="setlist-icon"
+            alt="setlist"
+            src={require('../icons/clipboard-list.svg')}
```
Comment (srmnq) [inline src/SideList/SideList.js]: better to import svg

File: src/SideList/SideList.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    setIsSetListsShown(true)
+    setIsAllSongsShown(false)
+    setIsASetListShown(false)
+  }
+
+  function handleAllSongsClick() {
+    setIsAllSongsShown(true)
+    setIsSetListsShown(false)
+    setIsASetListShown(false)
+    setSwipeIndex(0)
```
Comment (srmnq) [inline src/SideList/SideList.js:83]: very similar names...

File: src/SideList/SideList.js [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    }
+  }
+
+  function handleIsASetListShown(setlist) {
+    if (setlist.songs) {
+      sideListContent = setlist.songs.map((setlistSong, index) => {
+        return (
+          <SongListItem
+            key={setlistSong._id}
+            song={setlistSong}
```
Comment (srmnq) [inline src/SideList/SideList.js]: might combine this function with handle is allsongs shown

File: src/SideList/SideList.js [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+  justify-self: stretch;
+  align-self: center;
+  list-style: none;
+  padding: var(--titlePadding);
+  height: 100%;
+  border-radius: 12px 12px 0 0;
+  background: #3f4a6d;
+  background-clip: padding-box;
+  overflow: scroll;
+  padding-bottom: 72px;
```
Comment (srmnq) [inline src/SideList/SideList.js]: do you need this

File: src/common/GlobelStyles.js [hunk 6]
```diff
@@ -5,7 +5,7 @@ export default createGlobalStyle`
 
   
   :root{
-    --listWidth: 180px;
+    ${'' /* --listWidth: 180px; */}
```
Comment (srmnq) [inline src/common/GlobelStyles.js]: delete comment

**Review by srmnq — APPROVED** [review]
nice nice

File: src/common/GlobelStyles.js (hunk 6 above)
Comment (anniviz) [reply src/common/GlobelStyles.js]: done

File: src/SideList/SideList.js (hunk 5 above)
Comment (anniviz) [reply src/SideList/SideList.js]: done

File: src/SideList/SideList.js (hunk 4 above)
Comment (anniviz) [reply src/SideList/SideList.js]: done

File: src/App.js (hunk 1 above)
Comment (anniviz) [reply src/App.js:88]: done

File: src/SideList/SideList.js (hunk 3 above)
Comment (anniviz) [reply src/SideList/SideList.js:83]: not done because i have no alternative yet...

**Pull request merged**